src/migrations/M00004.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex) {
+	return knex('deviceConfig')
+		.select('targetValues')
+		.then(([target]) => {
+			const targetValues = target.targetValues;
+			const jsonValues = JSON.parse(targetValues);
+
+			if (jsonValues['SUPERVISOR_DELTA_APPLY_TIMEOUT'] == '') {
```

Comment:
Why don't we want strict typing?  What other values are you trying to match against? `0`/`[]`/`false`/?